src/components/NcAvatar/NcAvatar.vue

```diff
@@ -976,9 +976,9 @@ export default {
 		// - ⏹️ Best-fit: the status icon is as large as possible without exceeding the avatar box
 		// See PR for math explanation: PR #6004
 		--avatar-status-size-orbital: calc(var(--avatar-size) * (1 - 1 / sqrt(2)));
-		// Limit the status icon size to the status of a small clickable avatar
+		// Limit the status icon size to the status of a 50px avatar
 		// Ideally avatars with a smaller should not be used with the status icon at all
-		--avatar-status-size-min: calc(var(--default-clickable-area) * (1 - 1 / sqrt(2)));
+		--avatar-status-size-min: calc(50px * (1 - 1 / sqrt(2)));
```

Comment:
This is more than 14px, why a random number like 50px?
I think it makes more sense to use either clickable area large which is a value you might choose for the avatar when not using magic numbers:

```suggestion
		--avatar-status-size-min: calc(var(--clickable-area-large) * (1 - 1 / sqrt(2)));
```

Or because you want to limit the emoji to a readable size, then you do not need the avatar size here at all but just use the min. accessible size:

```suggestion
		--avatar-status-size-min: var(--font-size-small);
```

Reply:
font size seems to be optimal